### PR TITLE
Add documentation of `packit scan-in-osh` CLI command

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
The `packit scan-in-osh` is newly documented.

### CODE REVIEW

File: docs/cli/scan-in-osh.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: scan-in-osh
+date: 2024-05-22
+sidebar_position: 41
+
+---
+
+# `packit scan-in-osh`
+
+Builds an SRPM file and performs a scan in [OpenScanHub])(https://openscanhub.fedoraproject.org/).
```
Comment (nforro) [inline docs/cli/scan-in-osh.md:10]:
```suggestion
Builds a SRPM file and performs a scan in [OpenScanHub](https://openscanhub.fedoraproject.org/).
```

File: docs/cli/scan-in-osh.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Help
+
+    Usage: packit scan-in-osh [OPTIONS] [PATH_OR_URL]
+
+      Perform a scan through OpenScanHub. You need a valid kerberos ticket and set
```
Comment (nforro) [inline docs/cli/scan-in-osh.md:16]:
```suggestion
      Perform a scan through OpenScanHub. You need a valid Kerberos ticket and set
```

File: docs/cli/scan-in-osh.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      can be found at https://fedoraproject.org/wiki/OpenScanHub.
+
+    Options:
+      -p, --package TEXT  Package to build, if more than one available, like in a
+                          monorepo configuration. Use it multiple times to select
+                          multiple packages.Defaults to all the packages listed
+                          inside the config.
+      --target TEXT       Chroot to build in. (defaults to 'fedora-
+                          rawhide-x86_64')
+      --base-srpm TEXT    Base SRPM to perform a differential build
```
Comment (nforro) [inline docs/cli/scan-in-osh.md:27]:
```suggestion
      --base-srpm TEXT    Base SRPM to perform a differential build against
```

**Review by nforro — APPROVED** [review]

File: docs/cli/scan-in-osh.md (hunk 2 above)
Comment (lachmanfrantisek) [reply docs/cli/scan-in-osh.md:16]: This needs to be fixed in the CLI to preserve this..;) I'll take care of this and the other suggestion.. Thanks!

Comment (lachmanfrantisek) [reply docs/cli/scan-in-osh.md:16]: Here it is: https://github.com/packit/packit/pull/2319

**Comment by lachmanfrantisek** [comment]
regate

**Pull request merged**